packages/scaffold/src/views/w3m-wallet-send-view/index.ts

```diff
@@ -78,6 +89,16 @@ export class W3mWalletSendView extends LitElement {
   }
 
   // -- Private ------------------------------------------- //
+
+  private async fetchNetworkPrice() {
+    await ConvertController.getNetworkTokenPrice()
```

Comment:
Maybe this should be abstracted from convert controller to somewhere more generic?
Cc @enesozturk

Reply:
Leaving this for now, but I guess we need to refactor this a bit in the future